tests/unit/crypto/test_keystore.py

```diff
@@ -241,6 +241,33 @@ def test_restore_keystore_from_mnemonic(tmpdir, mocker):
     assert keystore._Keystore__secret == secret
 
 
+def test_import_custom_keystore(tmpdir):
+
+    # Too short - 32 bytes is required
+    custom_secret = b'tooshort'
+    with pytest.raises(ValueError, match="Entropy must be at least 32 bytes."):
```

Comment:
Shouldn't we also raise an error if it's more than 32 bytes? Currently any size from 32 to 64 bytes will work, and for anything over 64 an error from the depths of Umbral will be raised.

Reply:
✔️